dashboard/src/services/date.js

```diff
@@ -101,7 +101,11 @@ export const getIsDayWithinHoursOffsetOfPeriod = (dayToTest, { referenceStartDay
   const startDate = dayjs(referenceStartDay).startOf('day').add(offsetHours, 'hour');
   const endDate = dayjs(referenceEndDay).startOf('day').add(1, 'day').add(offsetHours, 'hour');
 
-  return dayjs(dayToTest).isBetween(startDate, endDate);
+  // Parameter 4 is a string with two characters; '[' means inclusive, '(' exclusive
+  // '()' excludes start and end date (default)
+  // '[]' includes start and end date
+  // '[)' includes the start date but excludes the stop
+  return dayjs(dayToTest).isBetween(startDate, endDate, null, '[]');
```

Comment:
Pourquoi il fallait changer ? en totalement inclusif (pas grave si tu sais plus)

```suggestion
  // Parameter 4 is a string with two characters; '[' means inclusive, '(' exclusive
  //  '()' excludes start and end date (default)
  //  '[]' includes start and end date
  //  '[)' includes the start date but excludes the stop
  // Source: https://day.js.org/docs/en/plugin/is-between
  return dayjs(dayToTest).isBetween(startDate, endDate, null, '[]');
```

Reply:
pour les passages anonymes, ils n'ont pas de date précise, donc je leur donne "start of day" comme date
et donc pour qu'ils soient inclus au bon compte rendu, il fallait que je mette en totalement inclusif

Reply:
Nickel !